### ISSUE TITLE
object_detection_tutorial.ipynb

### ISSUE THREAD

**Issue by ruhul-uni** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#589]
![Capture](https://user-images.githubusercontent.com/99887059/154728303-56c467d7-4072-4bd2-85fb-851fb051e557.PNG)

i get this issue but cannot find the file that is missing

**Comment by Petros626** [comment]
yeah because i think in the tutorial the wrong path is given but its in the folder structure from tensorflow

**Comment by prayagpawar** [comment]
jupyter notebook object_detection_tutorial.ipynb  is located in /models/research/object_detection/colab_tutorials
